### ISSUE TITLE
Member Sign-In and Hours Tracker

### ISSUE THREAD

**Issue by sidves52** [team-9143/team-website#4]
Incorporate member page to website with hours tracker, meeting rsvp, agendas, and todo list

See https://signin.team4159.org/

**Comment by totallytavi** [comment]
This requires a backend to be built. As it stands right now, this repository only works as a GitHub pages

**Comment by sidves52** [comment]
Right, hence why its tabled for now until we have enough time to go in and do it.
